clang/test/3C/base_subdir/unwritable_rootcauses.c

```diff
@@ -0,0 +1,7 @@
+// RUN: cd %S
+// RUN: 3c -use-malloc=my_malloc -alltypes -addcr -output-dir=%t.checked/base_subdir -warn-all-root-cause %s -- -Xclang -verify=unwritable-expected -Wno-everything
+
+#include "../root_cause.c"
+
+void foo(void) { 
+}
```

Comment:
Delete this empty function now that you've decided not to add back the code that manipulates the affected pointer counts?